app/src/components/fields/dynamicFieldsSection/utils.js

```diff
@@ -67,9 +67,9 @@ export const getFieldComponent = (field) => {
     fieldType = DATE_TYPE;
   } else if (field.definedValues && field.definedValues.length && field.fieldType !== ARRAY_TYPE) {
     fieldType = DROPDOWN_TYPE;
-  } else if (field.fieldType === AUTOCOMPLETE_TYPE && field.url) {
+  } else if (field.fieldType === AUTOCOMPLETE_TYPE && field.commandName) {
```

Comment:
My suggestion is to combine these who esle if

`else if(field.commandName){
    if(field.fieldType === AUTOCOMPLETE_TYPE){
         fieldType = AUTOCOMPLETE_TYPE
    }  
    if(field.fieldType === MULTIPLE_AUTOCOMPLETE_TYPE){
       fieldType = MULTIPLE_AUTOCOMPLETE_TYPE
    }
}`